commands/maliciousscan/maliciousscan.go

```diff
@@ -0,0 +1,282 @@
+	}
+	if scanner == nil {
+		return errors.New("JAS scanner was not created")
+	}
+
+	if cmd.customAnalyzerManagerPath != "" {
+		scanner.AnalyzerManager.AnalyzerManagerFullPath = cmd.customAnalyzerManagerPath
+	} else {
+		if scanner.AnalyzerManager.AnalyzerManagerFullPath, err = jas.GetAnalyzerManagerExecutable(); err != nil {
+			return fmt.Errorf("failed to set analyzer manager executable path: %w", err)
+		}
+	}
+
+	log.Debug(fmt.Sprintf("Using analyzer manager executable at: %s", scanner.AnalyzerManager.AnalyzerManagerFullPath))
+
+	jasScanProducerConsumer := utils.NewSecurityParallelRunner(cmd.threads)
+
+	serverDetails, err := cmd.ServerDetails()
+	if err != nil {
+		return err
+	}
+
+	jasScanProducerConsumer.JasWg.Add(1)
+	createMaliciousScansTask := func(threadId int) (generalError error) {
+		defer func() {
+			jasScanProducerConsumer.JasWg.Done()
+		}()
+		for _, targetResult := range cmdResults.Targets {
+			if targetResult.AppsConfigModule == nil {
+				_ = targetResult.AddTargetError(fmt.Errorf("can't find module for path %s", targetResult.Target), false)
+				continue
+			}
+			appsConfigModule := *targetResult.AppsConfigModule
+			jasParams := runner.JasRunnerParams{
+				Runner:         &jasScanProducerConsumer,
+				ServerDetails:  serverDetails,
+				Scanner:        scanner,
+				Module:         appsConfigModule,
+				ScansToPerform: []utils.SubScanType{utils.MaliciousCodeScan},
+				ScanResults:    targetResult,
+				TargetCount:    len(cmdResults.Targets),
+			}
+
+			if generalError = runner.AddJasScannersTasks(jasParams); generalError != nil {
+				_ = targetResult.AddTargetError(fmt.Errorf("failed to add malicious scan task: %w", generalError), false)
+				generalError = nil
+			}
+		}
+		return
+	}
+
+	if _, addTaskErr := jasScanProducerConsumer.Runner.AddTaskWithError(createMaliciousScansTask, func(taskErr error) {
+		cmdResults.AddGeneralError(fmt.Errorf("failed while adding JAS scan tasks: %s", taskErr.Error()), false)
+	}); addTaskErr != nil {
+		return fmt.Errorf("failed to create JAS task: %w", addTaskErr)
+	}
+
+	jasScanProducerConsumer.Start()
+
+	if cmd.progress != nil {
+		if err = cmd.progress.Quit(); err != nil {
+			return err
+		}
+	}
+
+	if err = output.NewResultsWriter(cmdResults).
+		SetOutputFormat(cmd.outputFormat).
+		SetPlatformUrl(cmd.serverDetails.Url).
+		SetPrintExtendedTable(false).
+		SetIsMultipleRootProject(cmdResults.HasMultipleTargets()).
+		SetSubScansPerformed([]utils.SubScanType{utils.MaliciousCodeScan}).
+		PrintScanResults(); err != nil {
+		return errors.Join(err, cmdResults.GetErrors())
+	}
+
+	if err = cmdResults.GetErrors(); err != nil {
+		return err
+	}
+
+	log.Info("Malicious scan completed successfully.")
+	return nil
+}
+
+func logScanPaths(workingDirs []string) {
+	if len(workingDirs) == 0 {
+		return
+	}
+	if len(workingDirs) == 1 {
+		log.Info("Scanning path:", workingDirs[0])
+		return
+	}
+	log.Info("Scanning paths:", strings.Join(workingDirs, ", "))
+}
```

Comment:
```suggestion
func logScanPaths(workingDirs []string) {
	if len(workingDirs) == 0 {
		return
	}
	if len(workingDirs) == 1 {
		log.Debug("Scanning path:", workingDirs[0])
		return
	}
	log.Debug("Scanning paths:", strings.Join(workingDirs, ", "))
}
```
I would move it to DEBUG since we already outputing targets later in Info